design-documents/testing/testing-tools.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+Magento actively leverages various testing strategies to ensure product and code quality. 
+
+Product quality matters to the end-users working with the Magento storefront, admin panel, web APIs and CLI. 
+Product quality tests verify that the user gets expected results while interacting with the system's interface.
+
+Code quality is important to developers working with Magento codebase including system integrators, extension developers, Magento core team and community contributors.
+Code quality matters for:
+ * Extensibility - how easy is it to extend or modify existing behavior.
+   Good extensibility allows for
+    * customizations and extensions using modularity of the platform
+    * evolution of the platform with new releases
+ * Maintainability - how easy is it to work with the system for developers. 
+   It can be improved with
+    * complexity management - reduces learning curve and risk of new bugs
+    * automatic bug detection - reduces total cost of ownership
+    * coding standards - reduces learning curve
+    
+Automated tests are required by [Magento definition of done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) for any code changes. 
+Most of the tests mentioned below are executed as part of the automated delivery process for each pull request before one is merged to the mainilne branch.
```

Comment:
I think instead of one it should be it. Would be good to ask tech writer to review.